tests/ci/run_fips_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+
+echo "Testing AWS-LC in FIPS debug mode."
+fips_build_and_test
+
+echo "Testing AWS-LC in FIPS release mode."
+fips_build_and_test -DCMAKE_BUILD_TYPE=Release
+
+if [[  "${AWSLC_NO_ASAM_FIPS}" == "1" ]]; then
```

Comment:
typo, should be AWSLC_NO_ASM_FIPS

Reply:
Fixed. If this PR gets two approvals, I will start the FIPS deployment.